Formula/libvirt.rb

```diff
@@ -14,6 +14,7 @@ class Libvirt < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gnutls"
+  depends_on "curl"
```

Comment:
Which version of `curl` does it require? I'd be surprised if the version of cURL on El Cap for example wasn't new enough.